### ISSUE TITLE
Request: Option to Duplicate Tab on Mobile

### ISSUE THREAD

**Issue by jonathansampson** [brave/brave-browser#22995]
## Description 

Users are able to easily duplicate a tab on desktop by right-clicking the tab and selecting _Duplicate_ from the context menu. Similar functionality has been requested by users for Android and iOS.

## Steps to reproduce 

When a tab is opened and a page is being viewed:

   1. Expand the page menu via ⋮ at the bottom-right
   2. Search for an option to Duplicate the tab

When viewing all tabs, with one selected:

   1. Expand the menu via ⋮ at the bottom-right
   2. Search for an option to Duplicate the selected/active tab

## Actual result 

No option exists to create a duplicate tab.

## Expected result

Users can easily duplicate their tab.

## Brave version

1.38.119, Chromium 101.0.4951.67

**Comment by joao-vitor-souza** [comment]
That's something I'd really like to see it implemented. It's kinda frustrating because I use this feauture on PC all the time, but when I use Brave on my phone I have to open a new tab, copy and paste the link, and yet the history of that tab is lost on the new tab.

**Comment by goyalyashpal** [comment]
vivaldi browser on android has this feature too for quite a long time already now.


<img src="https://github.com/user-attachments/assets/66a28d38-9494-4805-99ec-87493365c5b0" width=350px />

**Comment by codey-badger** [comment]
#46434 #50565 related issues, possibly duplicates.

**Comment by goyalyashpal** [comment]
yes, those newer duulicate issues can be closed in favour of this one.
